package.json

```diff
@@ -75,16 +74,17 @@
     "emoji-mart": "^2.11.2",
     "filesize": "3.6.1",
     "firstline": "1.2.1",
+    "fs": "^0.0.1-security",
```

Comment:
no need to add this here. You can just use readFileSync from fs-extra
`import { readFileSync } from 'fs-extra';`

Reply:
Noted. Thank you.